docker/package/tezos_setup_wizard.py

```diff
@@ -382,12 +382,18 @@ def import_baker_key(self):
 
             key_mode_query = get_key_mode_query(key_import_modes)
 
-            self.import_key(key_mode_query, "Baking")
+            not_set_up = True
```

Comment:
Even though it's minor here, IMO it reads better if there are no negations in variable names.

```suggestion
            ledger_set_up = False
```